README.md

```diff
@@ -1,30 +1,173 @@
 # RefArchSetup
 
 ## Overview
+RefArchSetup is a Ruby gem designed to help install the various Puppet Reference Architectures. 
+It currently supports the Standard Reference Architecture.
 
-RefArchSetup is a gem designed to help install the various puppet reference architectures
+# Prerequisites
+## Ruby
+RefArchSetup uses [Puppet Bolt](https://puppet.com/products/puppet-bolt) as a gem which requires a minimum Ruby version of 2.3.
+
+## Root Access
+RefArchSetup executes Bolt commands as the root user using the `--run-as root` option to ensure a successful PE installation. 
+See the [Bolt Options](#bolt-options) section for more information.
+
+## Supported Platforms
+RefArchSetup supports the following platforms:
+
+| OS                                                  | Arch   |
+|:----                                                |:----   |
+| EL (RHEL, CentOS, Scientific Linux, Oracle Linux) 6 | x86_64 |
+| EL (RHEL, CentOS, Scientific Linux, Oracle Linux) 7 | x86_64 |
+| SLES 12                                             | x86_64 |
+| Ubuntu 16.04                                        | amd64  |
+| Ubuntu 18.04                                        | amd64  |
+
+## Supported PE Versions
+RefArchSetup supports PE versions greater than 2018.1.0.
+
+## Supported Reference Architectures
+RefArchSetup currently supports the Standard Architecture.
 
 # Installation
+RefArchSetup can be installed via [RubyGems](https://rubygems.org/gems/ref_arch_setup) or by building the gem locally. 
 
+## Install via RubyGems
+The easiest way to install RefArchSetup is via [RubyGems](https://rubygems.org/gems/ref_arch_setup):
+```
     $ gem install ref_arch_setup
+```
+
+## Build the gem locally
+To build the gem locally:
+* Clone the RefArchSetup repository and install the dependencies by following the steps in the [Getting Started](CONTRIBUTING.md) section in [CONTRIBUTING.md](CONTRIBUTING.md).
+* From your local copy of the repository, build the gem using the provided rake task:
+```
+    $ ~/ref_arch_setup> bundle exec rake gem:build
+```
+* The gem will be built to the pkg directory; install the gem by either specifying the path to the RAS gem provided in the output from the previous step:
+```
+    $ ~/ref_arch_setup> gem install pkg/ref_arch_setup-0.0.x
+```
+or navigate to the pkg directory first, in which case specifying the version is not required:
+```
+    $ ~/ref_arch_setup> cd pkg && gem install ref_arch_setup
+```
 
 # Usage
+## Help
```

Comment:
I think under usage is a good place to note that only install bootstrap actually does anything, and that running install will call the others which will do noop.
Also, for the future, we probably don't want to put so much usage info in here... cause then we have to keep it in sync which is a pain.  Some basic examples of how to run it should be enough, and less effort to keep in sync.  But since it is here, it is fine for now.